node_groups.tf

```diff
@@ -420,6 +421,7 @@ module "self_managed_node_group" {
   enable_monitoring                  = try(each.value.enable_monitoring, var.self_managed_node_group_defaults.enable_monitoring, true)
   network_interfaces                 = try(each.value.network_interfaces, var.self_managed_node_group_defaults.network_interfaces, [])
   placement                          = try(each.value.placement, var.self_managed_node_group_defaults.placement, null)
+  launch_template_tags               = try(each.value.launch_template_tags, var.self_managed_node_group_defaults.launch_template_tags, null)
```

Comment:
likewise here, default of `{}` and move up to line 394

Reply:
done